prisma/prisma.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Global, Module } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+
+@Global()
+@Module({
+	providers: [PrismaService],
```

Comment:
em sửa lại indent cho toàn bộ project là 2 khoảng trắng thôi nhé, code javascript thường set indent như vậy.
chắc là text editor hoặc IDE nào cũng có phần cài đặt để custom cái này